src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -108,21 +122,25 @@ const SetAclPermsDocContainerForm = () => {
 
   return (
     <FormSection
-      title="Permission to Documents Container"
+      title={
+        user === 'personal' ? 'Set Permission to Documents' : 'Request Permissions to Documents'
```

Comment:
We haven't decided how to do request permissions yet. This will confuse people new to the project, and make them think we've made a decision already.

Reply:
We could add a comment on the set permission forms. As you mentioned, they're not finalized.